### PR TITLE
add about team page

### PR DESCRIPTION
**Pull request by kne42**
# Description

Add an about team page based on [this](https://docs.google.com/document/d/1MDijuDJyTAn3tu2dA7mdzA9NJ7-_JMtRMJtGzpXoAno/edit) copy.

preview of page

### CODE REVIEW

**Review by tlambert03 — APPROVED** [review]

**Review by Carreau — APPROVED** [review]

File: docs/community/team.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# About Team
+
+napari is a consensus-based community project. Anyone with an interest in the project can join the community, contribute to the project design, and participate in the decision making process. You can learn more about how the project is run by reading our [governance model](https://napari.org/community/governance.html). This page lists our current and emeritus [core developers](https://napari.org/community/governance.html#core-developers). Core developers are community members that have demonstrated a sustained commitment to the project through ongoing contributions and that they can maintain napari with care.
```
Comment (goanpeca) [inline docs/community/team.md:3]: Great work @kne42 

Question, should the `@kne42` be links and point to the github profiles?

like

```md
- [Kira Evans](https://github.com/napari/napari/commits?author=kne42) - [@kne42](https://github.com/kne42)
```

**Review by goanpeca — COMMENTED** [review]
Great work @kne42 

Question, should the `@kne42` be links and point to the github profiles?

like

```md
- [Kira Evans](https://github.com/napari/napari/commits?author=kne42) - [@kne42](https://github.com/kne42)
```

**Review by jni — APPROVED** [review]
I like @goanpeca's suggestion (links to gh profile). I'm also wondering why the file is not simply in the place where we want it, instead of having the .include file?

But those are minor, approving, feel free to self-merge when ready @kne42!

**Comment by sofroniewn** [comment]
Looking great @kne42 

> Question, should the @kne42 be links and point to the github profiles?

I like that idea too! Thanks for the review all!!

**Review by sofroniewn — APPROVED** [review]

**Comment by sofroniewn** [comment]
cc @napari/core-devs we're adding an "about team page" as the core-devs team is not public.

Once done we can fix #1190 and #1868 too (maybe @kne42 you can do that in a follow up PR or add to this one, thanks!!)

**Pull request merged**